map/src/context/RoutingManager.js

```diff
@@ -102,60 +103,439 @@ function filterMode(data) {
     return Object.fromEntries(Object.entries(data).filter(([key]) => !key.includes('rescuetrack')));
 }
 
+function osrmToFeaturesCollection(osrm) {
+/*
+    convert OSRM routes[] to features[] "LineString"
+    convert OSRM legs[].steps[] to features[] "Point" (OSRM's turns)
+    convert OSRM waypoints[] to features[] "Point" (OSRM's start-end)
+    convert OSRM duration/distance to properties
+
+    now support only 1st of routes[]
+*/
+    const features = [];
+
+    const cap = (s) => s && s[0].toUpperCase() + s.slice(1);
+
+    const maneuver = (s) => {
```

Comment:
rename to turns

Reply:
refused: 1) original naming from OSRM has "maneuver" 2) maneuvers is NOT turns only: there is Exit, Continue, New road, etc